src/TodoList.js

```diff
@@ -1,35 +1,18 @@
 import React from "react";
 import TodoListItem from "./TodoListItem";
-
-const todoList = [
-    {
-      id: 1,
-      title: "Wake up",
-    },
-    {
-      id: 2,
-      title: "Take a shower",
-    },
-    {
-      id: 3,
-      title: "Take a coffee",
-    },
-    {
-      id: 4,
-      title: "Learn React",
-    },
-  ];
   
-const TodoList = () => {
-return (
+function TodoList({ todoList }) {
+  return (
     <>
-    <ul>
-        {todoList.map((item) => (
-          <TodoListItem key={item.id} todo={item} />
-        ))}
-    </ul>
+      <ul>
+        {todoList.map(function (item) {
+          return (
+            <TodoListItem key={item.id} todo={item} />
+          );
+        })}
```

Comment:
This is fine and works perfectly well, but it's worth mentioning that you can also use destructuring here. Also same curiosity re: refactoring the arrow function out.

```suggestion
        {todoList.map(function ({ id, title }) {
          return (
            <TodoListItem key={id} todo={title} />
          );
        })}
```